chore: add initialization script for Reddit data

aleos-dev opened the pull request:
#117 
This commit includes an SQL script for initializing a set of sample data on Reddit. The script truncates existing data and resets identity sequences for the "user", "subreddit", "post", and "comment" tables. Then, it populates these tables with sample data representative of real-world usage. The aim is to provide a consistent test data set for the application.

aleos-dev committed:
chore: add initialization script for Reddit data

This commit includes an SQL script for initializing a set of sample data on Reddit. The script truncates existing data and resets identity sequences for the "user", "subreddit", "post", and "comment" tables. Then, it populates these tables with sample data representative of real-world usage. The aim is to provide a consistent test data set for the application.

Signed-off-by: Aleksei Os. <oal.aleos@gmail.com>

Y-Maxym commented on src/main/resources/init_reddit.sql:
Great, just please name it V3__data.sql. and put it in the db/migrations folder

aleos-dev replied:
Why is it version 3 when the development branch has version 1?

Y-Maxym replied:
You need a new version each time, otherwise flyway won't understand what to migrate after. I added V2 in one of the previous tasks, so the next one should be V3.

aleos-dev committed:
Move sql initialization file to migration folder

The init_reddit.sql has been renamed and moved to the db/migration folder. The new name is V3__data.sql, indicating its role and order in the database migration process.

Signed-off-by: Aleksei Os. <oal.aleos@gmail.com>
